docker/Dockerfile

```diff
@@ -28,6 +28,12 @@ RUN pip install future setuptools wheel clang flake8 && \
 
 RUN ldconfig
 
+RUN nvprune -gencode arch=compute_35,code=sm_35 -gencode arch=compute_50,code=sm_50 \
```

Comment:
Can we do the same with `libcufft_static.a`?

Reply:
I'm sorry but not really. It is more complicated.